Dockerfile-dev

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:22.04
```

Comment:
wouldnt it make sense to extend the non-dev one and just reinstall the packages with -e?